RampUpAuto/src/test/java/seleautomation/com/RampUpAuto/GitHubSighInTest.java

```diff
@@ -0,0 +1,145 @@
+package com.Github;
+
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.edge.EdgeDriver;
+import org.testng.annotations.*;
+import org.testng.Assert;
+
+
+class GitHubSignInPage {
+
+    WebDriver driver;
+
+    GitHubSignInPage(WebDriver driver) {
+
+        this.driver = driver;
+
+    }
+
+    public void signIn(String username, String password) {
+
+        WebElement usernameInput = driver.findElement(By.id("login_field"));
+
+        WebElement passwordInput = driver.findElement(By.id("password"));
+
+        WebElement signInButton = driver.findElement(By.name("commit"));
+
+        usernameInput.sendKeys(username);
+
+        passwordInput.sendKeys(password);
+
+        signInButton.click();
+
+    }
+
+    public boolean isSignInSuccessful() {
+
+        WebElement userAvatar = driver.findElement(By.className("avatar"));
+
+        return userAvatar.isDisplayed();
+
+    }
+
+    public boolean isErrorMessageDisplayed() {
+
+        WebElement errorMessage = driver.findElement(By.cssSelector(".flash-error"));
+
+        return errorMessage.isDisplayed();
+
+    }
+
+
+}
+
+public class GitHubSignInTest {
+    WebDriver driver;
+    public GitHubSignInPage signInPage;
+
+
+    @BeforeTest
+    @Parameters("browser")
```

Comment:
Regarding multiple running browser , how did you run it ? I mean did you control it through XML file or other technique because in this PR you just pushed this file

Reply:
I added those parameters to file testng.xml

<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite name="Default Suite">
  <test thread-count="5" name="ChromeTest">
    <parameter name="browser" value="chrome"></parameter>
    <classes>
      <class name="com.Github.GitHubSignInTest">
      </class> <!-- com.Github.GitHubSignInTest -->
    </classes>
  </test> <!-- GithubSigninProject -->

  <test thread-count="5" name="FireFoxTest">
    <parameter name="browser" value="firefox"></parameter>
    <classes>
      <class name="com.Github.GitHubSignInTest">
      </class> <!-- com.Github.GitHubSignInTest -->
    </classes>
  </test> <!-- GithubSigninProject -->

  <test thread-count="5" name="EdgeTest">
    <parameter name="browser" value="edge"></parameter>
    <classes>
      <class name="com.Github.GitHubSignInTest">
      </class> <!-- com.Github.GitHubSignInTest -->
    </classes>
  </test> <!-- GithubSigninProject -->
</suite> <!-- Default Suite -->